docs/sort.md

```diff
@@ -0,0 +1,66 @@
+# `vembrane sort`
+
+The `sort` subcommand allows to sort VCF/BCF files via keys defined as Python expressions (in ascending order).
+The Python expressions are analogous to expressions in other vembrane commands.
+This feature loads the entire VCF file into memory in order to maximize performance.
+It is thus meant to sort small, already filtered VCF files, e.g. for **prioritizing records for the human eye**.
+For large VCF files, the only relevant sorting is usually by position, which is better done with e.g. bcftools (and usually the sorting that variant callers output).
+
+### Usage
+
+```
+usage: vembrane sort [-h] [--output OUTPUT] [--output-fmt {vcf,bcf,uncompressed-bcf}] [--preserve-annotation-order]
+                     [--chunk-size CHUNK_SIZE] [--annotation-key FIELDNAME] [--aux NAME=PATH] [--ontology PATH]
+                     [--overwrite-number-info FIELD=NUMBER] [--overwrite-number-format FIELD=NUMBER]
+                     [--backend {cyvcf2,pysam}]
+                     [vcf] expression
+
+Sort VCF records by one or multiple Python expressions that encode keys for the desired order. This feature loads the entire
+VCF file into memory in order to maximize performance. It is thus meant to sort small, already filtered VCF files, e.g. for
```

Comment:
now that there's the external sort / --chunk-size option, no longer completely true